### ISSUE TITLE
Hands-Free Profile

### ISSUE THREAD

**Issue by elshnkhll** [pschatzmann/arduino-audio-tools#117]
Would it be too hard to make I2S-HFP (Hands-Free Profile, BT) and HFP-I2S examples? Thanks.

**Comment by pschatzmann** [comment]
Yes, I think that would be a project of it's own. Fortunatly there is an [example from Expressif](https://github.com/espressif/esp-idf/blob/master/examples/bluetooth/bluedroid/classic_bt/hfp_hf/README.md) that can be used as a basis
